spec/50-privacy.md

```diff
@@ -1,21 +1,23 @@
 # Privacy
 
-Requirements to propagate headers to downstream services as well as storing values of these headers opens up potential privacy concerns. Trace vendors MUST NOT use `traceparent` and `tracestate` fields for any personally identifiable or otherwise sensitive information. The only purpose of these fields is to enable telemetry correlation.
+Requirements to propagate headers to downstream services as well as storing values of these headers opens up potential privacy concerns. Trace vendors MUST NOT use `traceparent` and `tracestate` fields for any personally identifiable or otherwise sensitive information. The only purpose of these fields is to enable trace correlation.
```

Comment:
I intentionally use `telemetry` as one may use different data modfel that doesn't define trace. I used `trace` (in most cases `distributed trace`) to indicate an end-to-end set of telemetry. I was trying to avoid the term transaction.

Reply:
Then we need to define what telemetry means.